packages/Webkul/Admin/src/Services/DashboardService.php

```diff
@@ -0,0 +1,280 @@
+<?php
+
+namespace Webkul\Admin\Services;
+
+use Illuminate\Database\Eloquent\Collection;
+use Illuminate\Support\Carbon;
+use Illuminate\Support\Collection as SupportCollection;
+use Webkul\Customer\Repositories\CustomerRepository;
+use Webkul\Product\Models\Product;
+use Webkul\Product\Repositories\ProductInventoryRepository;
+use Webkul\Product\Repositories\ProductRepository;
+use Webkul\Sales\Models\OrderItem;
+use Webkul\Sales\Repositories\InvoiceRepository;
+use Webkul\Sales\Repositories\OrderItemRepository;
+use Webkul\Sales\Repositories\OrderRepository;
+
+class DashboardService
+{
+    protected Carbon $startDate;
+    protected Carbon $endDate;
+
+    protected Carbon $lastStartDate;
+    protected Carbon $lastEndDate;
+
+    /**
+     * @param OrderRepository $orderRepository
+     * @param OrderItemRepository $orderItemRepository
+     * @param InvoiceRepository $invoiceRepository
+     * @param CustomerRepository $customerRepository
+     * @param ProductInventoryRepository $productInventoryRepository
+     * @param ProductRepository $productRepository
+     */
+    public function __construct(
+        protected OrderRepository $orderRepository,
+        protected OrderItemRepository $orderItemRepository,
+        protected InvoiceRepository $invoiceRepository,
+        protected CustomerRepository $customerRepository,
+        protected ProductInventoryRepository $productInventoryRepository,
+        protected ProductRepository $productRepository,
+    )
+    {
+        $this->setLastStartDate();
+        $this->setLastEndDate();
+    }
+
+    /**
+     * @return array
```

Comment:
Remove the return type and just add a single line comment. As method is already strict type.